### DISCUSSION TITLE
Same git json attribute x_github_delivery for different webhook events

### DISCUSSION THREAD

**Discussion by gsm23** [community/community#24759] [Enterprise]
<p>I see multiple “x_github_delivery” with the same value. Further look into the JSON attribute I found commit ID also same.</p>
<p>However, "serialize_ts " is unique for all these events.</p>
<p>My question is at what condition above could happened?</p>
<p>I was assuming “x_github_delivery” must contain unique value for each event.</p>

**Comment by nethgato** [comment]
<p>Hi there <a class="mention" href="/gsm23">@gsm23</a> :wave:</p>
<p>Off the top, one possible answer could be multiple endpoints subscribed to the same webhook.</p>
<p>Are you able to let us know which repository you’re working from? If it’s private, I can initiate DM so I can do a bit of digging/reproduction on a repo of my own.</p>

**Comment by gsm23** [comment]
<p>Thanks for your response.</p>
<p>My code is in enterprise GitHub.</p>
<p>I have defined the webhook at Org level and it is the only endpoint configured.</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/gsm23">@gsm23</a> thanks so much for clarifying! Since you’re working with GH Enterprise, I would recommend that you reach out directly to our Enterprise Support team:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://enterprise.github.com/support" target="_blank" rel="noopener">enterprise.github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/62281706/181086596-dfe11f42-255f-40ff-9694-cddad9727949.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d906e4c1a04a608e211bb6dba332357009de24be.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_10x10.png"></div>

<h3><a href="https://enterprise.github.com/support" target="_blank" rel="noopener">Support - GitHub Enterprise</a></h3>

<p>The power of GitHub's social coding for your own workgroup. Pricing, tour and more.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>They should be able to help out!</p>
